_providers/163.md

```diff
@@ -12,6 +12,8 @@ server:
   socket: SSL
   hostname: smtp.163.com
   port: 465
-last_checked: 2022-06
+last_checked: 2024-11
```

Comment:
`PREPARATION` always requires a `before_login_hint`, sth. as:

```suggestion
last_checked: 2024-11
before_login_hint: Enable "POP3/SMTP/IMAP" on the website, add a third-party auth code and use that as the login password
```

below the text a "Read More" link will be shown automatically. that link then opens https://providers.delta.chat/163